.travis.yml

```diff
@@ -11,8 +11,9 @@ env:
   - TEST_DIR=runtime
 cache:
   pip: true
+  yarn: true
```

Comment:
I don't think this works, as the language used for Travis is `python`; that's why the yarn directory was previously explicitly spelled out.

Reply:
https://blog.travis-ci.com/2016-11-21-travis-ci-now-supports-yarn

Reply:
I know they've made that post, but that only applies if your language is `node_js`; if it's not, then the `yarn: true` does nothing.

Before change: https://travis-ci.org/pioneers/PieCentral/jobs/294230172
After change: https://travis-ci.org/pioneers/PieCentral/jobs/294243830

Note that the
```
adding /home/travis/.cache/yarn to cache
creating directory /home/travis/.cache/yarn
```
lines have disappeared in "after change" build logs.